guidelines.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Package Guidelines
+description: Guidelines for Pangeo packages to follow
+---
+
+Our vision for the Pangeo project is an ecosystem of mutually compatible
+Geoscience python packages which follow open-source best practices.
+These practices are well established across the scientific python community.
+
+
+### General Best Practices for Open Source
+
+1. Open source license
+1. Community guidelines (provide example)
+1. Version control of source code (for example, on [github](http://github.org))
+1. Thorough test coverage and continuous integration of testing
+1. Comprehensive Documentation
+
+
+### Best Practices for Pangeo Projects
+
+To address the needs of geoscience researchers, we have developed some
+additional recommendations.
+
+1. _Solve a general problem:_ packages should solve a general problem
+that is encountered by broad groups of researchers.
+1. _Clearly defined scope:_ packages should have a clear and relatively
+narrow scope.
+1. _No duplication:_ developers should try to leverage existing
+packages as much as possible to avoid duplication of effort.
+1. _Consume and Produce XArray Objects:_ XArray data structures facilitate
+mutual interoperability between packages.
+1. _Operate Lazily:_ packages should avoid explicitly triggering computation
+on [Dask](http://dask.pydata.org/en/latest/array.html) objects.
+
```

Comment:
This is a great point. (Also applies to things like seawater eqn of state.)

My hope is that xarray.apply will one day be able to handle this sort of module (in a lazy way).